repose-aggregator/src/docs/asciidoc/services/datastores.adoc

```diff
@@ -52,53 +48,47 @@ The following is a basic sample configuration.
 ----
 
 ==== Full Configuration
-
 [source,xml]
 .dist-datastore.cfg.xml
 ----
 <?xml version="1.0" encoding="UTF-8"?>
 <distributed-datastore xmlns='http://docs.openrepose.org/repose/distributed-datastore/v1.0'
-                       connection-pool-id="default"         <!--1-->
-                       keystore-filename="keystore.jks"     <!--2-->
-                       keystore-password="password"         <!--3-->
-                       key-password="secret"                <!--4-->
+                       connection-pool-id="default" <!--1-->
+                       keystore-filename="keystore.jks" <!--2-->
+                       keystore-password="password" <!--3-->
+                       key-password="secret" <!--4-->
                        truststore-filename="truststore.jks" <!--5-->
-                       truststore-password="trusting">      <!--6-->
-    <allowed-hosts allow-all="false">                       <!--7--><!--8-->
-        <allow host="127.0.0.1"/>                           <!--9-->
+                       truststore-password="trusting"> <!--6-->
+    <allowed-hosts allow-all="false"> <!--7--> <!--8-->
+        <allow host="127.0.0.1"/> <!--9-->
     </allowed-hosts>
     <port-config>
-        <port port="9999" cluster="repose"/>                <!--10-->
-        <port port="7777" cluster="repose" node="node2"/>   <!--11-->
+        <port port="9999" cluster="repose"/> <!--10-->
+        <port port="7777" cluster="repose" node="node2"/> <!--11-->
     </port-config>
 </distributed-datastore>
 ----
 <1> HTTP Connection pool (ID) to use when communicating with other members of the distributed datastore.
 <2> IF this attribute is configured,
     THEN it is assumed that it points to the Java keystore containing the client certificate to present for client authentication (e.g keystore.jks)
     AND the `keystore-password` and `key-password` attributes are no longer optional. +
-    [NOTE]
-    ====
-    IF the `keystore-filename` is defined,
-    THEN HTTPS will be used;
-    ELSE HTTP will be used.
-    ====
+    *NOTE:* +
+    _IF_ the `keystore-filename` is defined, +
+    _THEN_ HTTPS will be used; +
+    _ELSE_ HTTP will be used.
```

Comment:
Oh wait. I guess bolding `NOTE:` prevents asciidoctor from making it an admonition.'

Fair enough!